src/Provider/Provider.Web/Configuration/Routing/RoutePaths.cs

```diff
@@ -3,6 +3,7 @@
     public static class RoutePaths
     {
         public const string AccountRoutePath = "{ukprn:length(8)}";
+        public const string VacanciesRoutePath = "vacancies/{ukprn:length(8)}";
```

Comment:
This should be `public const string VacanciesRoutePath = AccountRoutePath +"/vacancies";`